ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/behaviorTree/BehaviorTreeDefinitionBuilder.java

```diff
@@ -49,7 +53,7 @@ public static BehaviorTreeNodeDefinition createNode(Class<?> definitionType, CRD
       }
       if (definitionType == FootstepPlanActionDefinition.class)
       {
-         return new FootstepPlanActionDefinition(crdtInfo, saveFileDirectory, new DefaultFootstepPlannerParameters());
+         return new FootstepPlanActionDefinition(crdtInfo, saveFileDirectory, new DefaultFootstepPlannerParameters(defaultFootstepPlannerParameters));
```

Comment:
Make a copy of the defaults to pass in. This instance is held onto and modified by the node.